scripts/plot_joined_scalars.py

```diff
@@ -0,0 +1,147 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+import os
+from collections import OrderedDict
+import oemoflex.tools.plots as plots
+
+q = "capacity"
+w = "flow_in_electricity_from_bus"
+e = "flow_in_electricity_to_bus"
+r = "flow_out_electricity_from_bus"
+t = "flow_out_electricity_to_bus"
+z = "transmission_losses"
+
+u = "flow_in_biomass"
+i = "flow_out_electricity"
+o = "invest_out_electricity"
+p = "summed_variable_costs_in_biomass"
+a = "summed_variable_costs_out_electricity"
+s = "flow_in_electricity"
+d = "invest"
+f = "invest_in_electricity"
+g = "storage_capacity"
+h = "storage_losses"
+j = "flow_in_ch4"
+k = "summed_variable_costs_in_ch4"
+
+var_name = q
+
+names_dict = {"BE-biomass-st":"Biomass","BB-biomass-st":"Biomass",
+              "BE-electricity-curtailment":"El. curtailment","BB-electricity-curtailment":"El. Curtailment",
+              "BE-electricity-demand":"El demand","BB-electricity-demand":"El. Demand",
+              "BE-electricity-liion-battery":"Liion Battery","BB-electricity-liion-battery":"Liion Battery",
+              "BE-electricity-shortage":"El. shortage","BB-electricity-shortage":"El. Shortage",
+              "BE-BB-electricity-transmission":"El. Transmission",
+              "BE-ch4-gt": "CH4", "BB-ch4-gt": "CH4",
+              "BE-solar-pv":"PV","BB-solar-pv":"PV",
+              "BE-wind-onshore":"Wind on","BB-wind-onshore":"Wind on",
+              }
+
+unit_dict = {"capacity":"W"}
+
+# load colors dict which also determines the ordering
+colors_csv = pd.read_csv("colors.csv", header=[0], index_col=[0])
+colors_csv = colors_csv.T
+colors_odict = OrderedDict()
+for i in colors_csv.columns:
+    colors_odict[i] = colors_csv.loc["Color", i]
```

Comment:
Here, you could use the colors defined in oemoflex.

```suggestion
from oemoflex.tools.plots import colors_odict
```